### PR TITLE
Update to cosmoshub-2

### PR DESCRIPTION
**Pull request by hendrikhofstadt**
We independently followed the guide and reviewed the migration script to generate the new genesis file for `cosmoshub-2`

### CODE REVIEW

**Comment by ebuchman** [comment]
Can we please not delete the original genesis.json and give this one the name from the guide, ie. `cosmoshub_1_genesis_export.json` ?

**Comment by mdyring** [comment]
Have verified the export / migration steps locally and confirm the sha256sum:

```
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30  genesis.json
```

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Cosmos Hub Upgrade 1
+
+**Transfer enablement**
+
+According to governance proposal #3 and #5, the state of the `cosmoshub-1` chain was exported at block 500.000 and migrated to `cosmoshub-2` using [this guide](https://github.com/cosmos/cosmos-sdk/wiki/Cosmos-Hub-1-Upgrade).
```
Comment (alexanderbez) [inline README.md]:
```suggestion
According to governance proposals #3 and #5, the state of the `cosmoshub-1` chain was exported at block 500.000 and migrated to `cosmoshub-2` using [this guide](https://github.com/cosmos/cosmos-sdk/wiki/Cosmos-Hub-1-Upgrade).
```

**Comment by hendrikhofstadt** [comment]
> Can we please not delete the original genesis.json, and then give this one the name from the guide, ie. `cosmoshub_1_genesis_export.json` ?
> 
> We can reorg this repo a bit afterwards to better accomodate this process and future upgrades

I moved the original genesis file to `genesis.cosmoshub-1.json` to preserve the old initial genesis.

Do you think the pure export is important as well ?

**Comment by ebuchman** [comment]
Oh I see I didn't quite catch that. So this doesn't even add the new genesis then?

**Comment by NodeATeam** [comment]
[export]
sha256sum  genesis.json
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30  genesis.json

[export + adding newlines]
sha256sum  genesis.json
4f641b020b4489c9e478ee35b648c3d5da439055c13c5062bbc8366f0e5b7901  genesis.json

**Comment by fkbenjamin** [comment]
Can confirm that this genesis file produces the same hash as with generating it locally.

```
shasum -a 256 certusGenesis.json
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30  certusGenesis.json
```

Let's get cosmoshub-2 started :)

**Comment by alexanderbez** [comment]
@ebuchman it does. `genesis.json` is the new genesis and `genesis.cosmoshub-1.json` is the old.

**Comment by JayB-K** [comment]
I can also confirm that this genesis file produces the same hash 👍 
```
shasum -a 256 genesis.json
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30 genesis.json
```

**Comment by winslyn** [comment]
I confirm from locally generated genesis.json
```
sha256sum genesis.json
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30  genesis.json
```

**Comment by gin** [comment]
👍 
```
$ sha256sum genesis.json 
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30  genesis.json
```

**Comment by liangping** [comment]
[cosmos@cosmos-sentry config]$ shasum -a 256 genesis.json
1e349fb39b85f7707ee78d39879f9d5d61f4d30f67980bb0bf07bd35b2f8bf30  genesis.json

**Pull request merged**